generators/app/templates/src/app/components/CustomText/index.js

```diff
@@ -25,9 +25,9 @@ export default class CustomText extends PureComponent {
   ];
 
   customStyles = () =>
-    CustomText.VARIANTS
-      .map(variant => (this.props[variant] ? styles[variant] : null))
-      .filter(style => style !== null);
+    CustomText.VARIANTS.map(variant => (this.props[variant] ? styles[variant] : null)).filter(
+      style => style !== null
+    );
```

Comment:
I'd reuse the function extracted from `CustomButton`

Reply:
👍